src/main/java/seedu/address/logic/commands/meeting/UpdateCommand.java

```diff
@@ -38,7 +38,7 @@
             + "[" + PREFIX_MEETING_TIME + "MEETING_TIME] "
             + "[" + PREFIX_MEETING_PLACE + "MEETING PLACE] ";
 
-    public static final String MESSAGE_UPDATE_MEETING_SUCCESS = "%1$s's information has been updated";
+    public static final String MESSAGE_UPDATE_MEETING_SUCCESS = "Meeting has been updated";
```

Comment:
This works fine, but I'm curious why we can't use the meeting agenda here?

Reply:
I tried that as well. What happens is when you update the meeting agenda the success message uses the old agenda which is kind of a confusing behaviour.